core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -9,3 +9,4 @@ registry.svc.ci.openshift.org/openshift-kni/4.5:sctptester quay.io/openshift-kni
 registry.svc.ci.openshift.org/openshift-kni/4.5:cnftest-utils quay.io/openshift-kni/cnftest-utils:4.5 quay.io/openshift-kni/cnftest-utils:4.5.0 quay.io/openshift-kni/cnftest-utils:latest
 registry.svc.ci.openshift.org/openshift-kni/4.5:stresser quay.io/openshift-kni/stresser:4.5 quay.io/openshift-kni/stresser:4.5.0 quay.io/openshift-kni/stresser:latest
 registry.svc.ci.openshift.org/openshift-kni/4.5:dpdk quay.io/openshift-kni/dpdk:4.5 quay.io/openshift-kni/dpdk:4.5.0 quay.io/openshift-kni/dpdk:latest
+registry.svc.ci.openshift.org/openshift-kni/4.6:cnf-tests quay.io/openshift-kni/cnf-tests:4.6 quay.io/openshift-kni/cnf-tests:4.6.0 quay.io/openshift-kni/cnf-tests:latest
```

Comment:
- `cnf-tests:latest` will be mirrored from 4.5.0 and 4.6.0 now, the last one wins. I don't think this is wanted?
- and what about the other images?

Maybe it's best to switch from using `latest` tag to versioned tags?

Reply:
>     * `cnf-tests:latest` will be mirrored from 4.5.0 and 4.6.0 now, the last one wins. I don't think this is wanted?

Not wanted at all!
> 
>     * and what about the other images?
> 
The others are used for tests. For now I'd use the 4.5 version even in 4.6 tests, at least for the initial phase.

> 
> Maybe it's best to switch from using `latest` tag to versioned tags?

Where we use the images we are already pinning to 4.5, if this is what you mean.

Reply:
> For now I'd use the 4.5 version even in 4.6 tests
> we are already pinning to 4.5

ah, I missed that we are pushing both latest and versioned images already. Just using 4.5. for now works for me :+1: